mixins/base-child.js

```diff
@@ -38,6 +38,25 @@ const baseChild = {
   },
 
   /**
+   * Creates a new metafield on a record.
+   * This also works to update the existing metafield
+   *
+   * @param {Number} parentId Parent record ID
+   * @param {Object} params Record properties
+   * @return {Promise} Promise that resolves with the result
+   * @public
+   */
+  createMetafield(parentId, params) {
```

Comment:
`Product` does not use this mixin so this method will not be available for `shopify.product`.